### PR TITLE
Fix null check

### PR DESCRIPTION
**Pull request by terminal-cs**

### CODE REVIEW

File: source/Cosmos.IL2CPU/AppAssembler.cs [hunk 1]
```diff
@@ -63,12 +63,8 @@ public AppAssembler(CosmosAssembler aAssembler, TextWriter aLog, string aLogDir)
 
         public void Dispose()
         {
-            if (mLog != null)
-            {
-                mLog.Dispose();
-                mLog = null;
-            }
-            DebugInfo.Dispose();
+            mLog?.Dispose();
```
Comment (quajak) [inline source/Cosmos.IL2CPU/AppAssembler.cs:66]: Why is mLog no longer set to null?

**Review by quajak — CHANGES_REQUESTED** [review]
See comment

File: source/Cosmos.IL2CPU/AppAssembler.cs (hunk 1 above)
Comment (terminal-cs) [reply source/Cosmos.IL2CPU/AppAssembler.cs:66]: Why would it need to be? There's no point in putting it to null, it doesn't free any extra memory

**Review by quajak — APPROVED** [review]
True, looking at the code it isn't required

**Pull request merged**